File: go/raindrops/raindrops.go
package raindrops

import (
    "strconv"
)

func Convert( num int ) string {
    var str string = ""

    if(num%3==0) {
        str += "Pling"
    }
    if(num%5==0) {
        str += "Plang"
    }
    if(num%7==0) {
        str += "Plong"
    }
    if(str == "") {
        return strconv.Itoa(num)
    }
    return str
}

File: python/raindrops/raindrops.py
def raindrops(number):
    numStr = ""
    if number%3 == 0:
        numStr += "Pling"
    if number%5 == 0:
        numStr += "Plang"
    if number%7 == 0:
        numStr += "Plong"
    if numStr == "":
        numStr = str(number)
    return numStr

